pkg/services/featuremgmt/registry.go

```diff
@@ -1337,6 +1337,24 @@ var (
 			Owner:       grafanaObservabilityMetricsSquad,
 			Created:     time.Date(2024, time.January, 29, 0, 0, 0, 0, time.UTC),
 		},
+		{
+			Name:         "slateAutocomplete",
+			Description:  "Adjusts the behaviour of the slate editor to properly handle autocomplete. Feature toggled for safety.",
+			Stage:        FeatureStageGeneralAvailability,
+			Expression:   "true", // enabled by default
+			FrontendOnly: true,
+			Owner:        grafanaFrontendPlatformSquad,
+			Created:      time.Date(2024, time.January, 29, 12, 0, 0, 0, time.UTC),
+		},
+		{
+			Name:         "sqlExpressions",
+			Description:  "Enables using SQL and DuckDB functions as Expressions.",
+			Stage:        FeatureStageExperimental,
+			Expression:   "true", // enabled by default
+			FrontendOnly: false,
+			Owner:        grafanaBiSquad,
+			Created:      time.Date(2024, time.January, 31, 12, 0, 0, 0, time.UTC),
```

Comment:
can we make this feature devMode only for now?  My biggest concern is some method for  rate/scale limiting in cloud as this is potentially a very expensive service!